README.rst

```diff
@@ -57,9 +57,9 @@ or by `joining our Gitter chat <https://gitter.im/LiberTEM/Lobby>`_.
 LiberTEM is suitable as a high-performance processing backend for other applications, including live data streams. `Contact us <https://gitter.im/LiberTEM/Lobby>`_ if you are interested! 
 
 
-LiberTEM is evolving rapidly and prioritizes features following user demand and contributions. In the future we'd like to implement live acquisition, and more analysis methods for all applications of pixelated STEM.
+LiberTEM is evolving rapidly and prioritizes features following user demand and contributions. In the future we'd like to implement live acquisition, and more analysis methods for all applications of pixelated STEM and other large-scale detector data.
 If you like to influence the direction this
-project is taking, or if you'd like to contribute, please join our `gitter chat <https://gitter.im/LiberTEM/Lobby>`_,
+project is taking, or if you'd like to :doc:`contribute <contributing>`, please join our `gitter chat <https://gitter.im/LiberTEM/Lobby>`_,
```

Comment:
Using roles like `:doc:` in the README.rst may be problematic, as they may not be supported by the pypi renderer.

See also https://github.com/LiberTEM/LiberTEM/commit/f01df221383970bc9edad438a88d83ad231ca43b - if you rebase to current master you should get a validation error in CI.